src/ansi-c/library/math.c

```diff
@@ -854,11 +899,14 @@ float sqrtf(float f)
     // number of exponent and significand bits.  Thus they are
     // given implicitly...
 
+#pragma CPROVER check push
+#pragma CPROVER check disable "float-overflow"
     float lowerSquare = lower * lower;
     __CPROVER_assume(__CPROVER_isnormalf(lowerSquare));
```

Comment:
A square root may well be denormal?

Reply:
I don't think that's possible for the case that `f` (which we are taking the square root of) is normal, which this branch assumes: given a denormal number is smaller than 1, any denormal square root `s` cannot be smaller than `f` (for $s < 1 \wedge s \geq 0 \implies s^2 \leq s$), so `f` has to be denormal when `s` is denormal.